src/sqlfluff/core/default_config.cfg

```diff
@@ -148,6 +148,9 @@ wildcard_policy = single
 # Trailing commas
 select_clause_trailing_comma = forbid
 
+[sqlfluff:rules:L039]
+align_alias = True
```

Comment:
I think you'll want to change this to `False` before merging.

Reply:
Ok, im running `pytest test` locally with `False` right now, and will update if things go smoothly